backend/tests/students.test.js

```diff
@@ -0,0 +1,77 @@
+import request from 'supertest';
+import app from '../app';
+import db, { sequelize } from '../models';
+// import Student from '../models/Student.model.js';
+
+const { Student } = db;
+
+describe('GET /api/students', () => {
+  beforeAll(async () => {
+    try {
+      console.log('Syncing database...');
+      await sequelize.sync({ force: true }); // Sync the database and clear existing data
+      console.log('Database synced successfully!');
+
+      await Student.bulkCreate([
+        {
+          name: 'John Doe',
+          email: 'johndoe@example.com',
+          student_id: '123456789',
+        },
+        {
+          name: 'Jane Doe',
+          email: 'janedoe@example.com',
+          student_id: '987654321',
+        },
+        {
+          name: 'Alice Smith',
+          email: 'alices@example.com',
+          student_id: '456789123',
+        },
+      ]);
+    } catch (error) {
+      console.error('Error connecting to the database:', error);
+    }
+  });
+
+  afterAll(async () => {
+    // Clean up the test database
+    try {
+      await sequelize.close();
+    } catch (error) {
+      console.error('Error closing the database:', error);
+    }
```

Comment:
Similar to my other comment, can you move this afterall to a global teardown? See https://jestjs.io/docs/configuration#globalsetup-string and https://jestjs.io/docs/configuration#globalsetup-string